projectDocs/translating/crowdin.md

```diff
@@ -1,45 +1,83 @@
+
+Note: If you are a screen reader user, you may find Crowdin's web interface inefficient or impossible to use. Therefore you may choose to use the Poedit workflow instead.
+
+#### Translation Reviews / Approvals
+Due to accessibility issues and to be able to support the alternative workflow using Poedit, the translation approvals feature has been disabled in the NVDA project on Crowdin.
+This means that all new strings are essentially auto approved.
+ However, to maintain quality control, only translators specifically added to the project can add or change strings.
+
+### Translating with Poedit
+Poedit is a desktop application which is commonly used to translate files.
+It is fairly accessible and is used by many blind and vision impaired translators.
+
+The workflow for translating with Poedit involves:
+1. Downloading the translation file using NVDA's l10n utility
+1. Opening the translation file in Poedit, translating one or more strings, and saving the file.
+1. Uploading the translation file back to Crowdin.
+
+Warning: Do not download / upload translation files via Crowdin's web interface, nor use Poedit's Crowdin cloud translation feature, as these methods have been found to corrupt xliff files.
+Please instead use NvDA's l10n utility for download and upload.
+NVDA's l10n utility will automatically use the appropriate Crowdin settings when downloading and uploading, and will detect and correct corruptions in xliff files.
+
+#### Setting up PoEdit
 
 It is recommended that you use the latest version of PoEdit and NVDA for translating.
-Alternatively, you can use the [Crowdin web interface](https://support.crowdin.com/online-editor/) directly.
-As PoEdit only supports viewing approved strings, large translator teams need to co-ordinate submitting unapproved strings to prevent conflicts.
-Using Crowdin's interface avoids this problem.
 
 PoEdit's homepage is: <http://www.poedit.net/>
 
 1. Download the latest Windows PoEdit version at <https://poedit.net/download>
 1. Install it by following the on-screen instructions, the default options should be sufficient.
 
-### Translation reviews
-Due to accessibility issues, for now translation approvals have been disabled on Crowdin.
-Any translation uploaded to Crowdin is automatically available in the project.
-However, joining the project as a translator is by invitation only.
+As PoEdit only supports viewing approved strings, large translator teams need to co-ordinate submitting unapproved strings to prevent conflicts.
+Using Crowdin's interface avoids this problem.
 
-## Translation workflows
+#### Locating the NVDA l10n Utility
+The NVDA l10n utility is required for safely and efficiently downloading and uploading translation files from / to Crowdin when translating with Poedit.
 
-There are 2 common workflows for translating with Crowdin:
+ This utility is included with all versions of NVDA from 2025.1beta1 onwards.
```

Comment:
I think it's better to not mention beta. 
Also - is this something we can upload separately like `controllerClient`?
I can look into uploading it so we can just link something like <https://download.nvaccess.org/releases/beta/l10Util.exe/> in future.

```suggestion
This utility is included with all versions of NVDA from 2025.1 onwards.
```

At least, strip the leading space

```suggestion
This utility is included with all versions of NVDA from 2025.1beta1 onwards.
```

Reply:
Unfortunately, as it is currently compiled with Py2exe as a part of our distribution, it cannot be decoupled from NvDA, as it depends on library.zip / and the Python dll etc.
For a while we did compile it stand-alone with nuitka, but that took like 16 minutes to compile on appveyor.
We could switch back to Nuitka, make it optional and compile once and upload somewhere... But in reality most if not all translators will be running NVDA.